src/System.Private.CoreLib/src/System/Diagnostics/Eventing/EventPipeController.cs

```diff
@@ -32,12 +34,19 @@ internal sealed class EventPipeController
     {
         // Miscellaneous constants.
         private const string NetPerfFileExtension = ".netperf";
-        private const string MarkerFileExtension = ".ctl";
+        private const string ConfigFileName = "EventPipeConfig.txt";
```

Comment:
Nit.  I am abivalent about *.txt.   Unix does not have suffixes, and this is not really a  text file anyway.   I am inclined to make it simply 'EventPipeConfig'.